api/src/main/java/org/openmrs/module/eptsreports/metadata/HivMetadata.java

```diff
@@ -949,6 +949,14 @@ public Concept getAbsenceOfHealthProviderInHealthUnitConcept() {
     return getConcept(uuid);
   }
 
+  // concept_id = 1410
```

Comment:
Hi @KelvinWanyama this  is in https://github.com/esaude/openmrs-module-eptsreports/blob/bcc18f88ea5ef727c09022a1ad7fd3616564e4ef/api/src/main/java/org/openmrs/module/eptsreports/metadata/CommonMetadata.java#L104  so we do not need to add in `HivMetadata.class`

Reply:
okay thanks.